requirements-dev.txt

```diff
@@ -1,3 +1,20 @@
+aiohttp==3.7.4.post0
```

Comment:
Looks like you copy-pasted the output of the `pip freeze` command.

It is considered a bad practice. Usually, you specify only libs you are using directly like `alembic`.  

I'd recommend having 2 files with requirements for the project: one with requirements to run the project (alembic, etc) and the other for development staff (tests, linter, formatters).  So for development, you need to install from both of them and for production only from one of them.